exasol/toolbox/util/git.py

```diff
@@ -0,0 +1,51 @@
+import subprocess  # nosec
+from functools import wraps
+from pathlib import Path
+
+
+def run_command(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs) -> str:
```

Comment:
I think this is a very nice and highly appreciated approach. 
But it only works as long as a command doesn't contain any spaces.
E.g. The approach will fail if using `"echo 'a b c' > file.txt"`

I would feel more save if using a list of string, I guess.